tests/test_functional.py

```diff
@@ -405,6 +405,43 @@ def test_award_one_medal(self, user, testapp, medal_id):
         assert getattr(do, "awarded_medal_" + medal.slug) is True
         assert AwardedMedals.query.get((do.rodne_cislo, medal.id)) is not None
 
+    @pytest.mark.parametrize("rodne_cislo", sample_of_rc(10))
+    def test_medal_amount(self, user, testapp, rodne_cislo):
+        do = DonorsOverview.query.get(rodne_cislo)
+        medals = Medals.query.all()
+        login(user, testapp)
+        res = testapp.get(url_for("donor.detail", rc=rodne_cislo))
+
+        medal_amount = 0
+        for medal_slug in [medal.slug for medal in medals]:
+            if getattr(do, "awarded_medal_" + medal_slug):
+                medal_amount += 1
```

Comment:
Udělené medaile lze zjistit i podstatně snáz.

```suggestion
        login(user, testapp)
        res = testapp.get(url_for("donor.detail", rc=rodne_cislo))
        medal_amount = AwardedMedals.query.filter(AwardedMedals.rodne_cislo == rodne_cislo).count()
```

Reply:
Upraveno